src/storage-preview/azext_storage_preview/operations/account.py

```diff
@@ -316,3 +316,86 @@ def update_file_service_properties(cmd, instance, enable_delete_retention=None,
+
+    if home_directory is not None:
+        local_user.home_directory = home_directory
+    if has_shared_key is not None:
+        local_user.has_shared_key = has_shared_key
+    if has_ssh_key is not None:
+        local_user.has_ssh_key = has_ssh_key
+    if has_ssh_password is not None:
+        local_user.has_ssh_password = has_ssh_password
+
+
+def create_local_user(cmd, client, resource_group_name, account_name, username, permission_scope=None, home_directory=None,
+                      has_shared_key=None, has_ssh_key=None, has_ssh_password=None, ssh_authorized_key=None, **kwargs):
+    LocalUser, PermissionScope, SshPublicKey = cmd.get_models('LocalUser', 'PermissionScope', 'SshPublicKey')
+    local_user = LocalUser()
+
+    _generate_local_user(local_user, PermissionScope, SshPublicKey, permission_scope, ssh_authorized_key,
+                         home_directory, has_shared_key, has_ssh_key, has_ssh_password)
+    return client.create_or_update(resource_group_name=resource_group_name, account_name=account_name,
+                                   username=username, properties=local_user)
+
+
+def update_local_user(cmd, client, resource_group_name, account_name, username, permission_scope=None,
+                      home_directory=None, has_shared_key=None, has_ssh_key=None, has_ssh_password=None,
+                      ssh_authorized_key=None, **kwargs):
+    local_user = client.get(resource_group_name, account_name, username)
+    PermissionScope, SshPublicKey = cmd.get_models('PermissionScope', 'SshPublicKey')
+
+    _generate_local_user(local_user, PermissionScope, SshPublicKey, permission_scope, ssh_authorized_key,
+                         home_directory, has_shared_key, has_ssh_key, has_ssh_password)
+    return client.create_or_update(resource_group_name=resource_group_name, account_name=account_name,
+                                   username=username, properties=local_user)
+
+
+def delete_local_user(client, resource_group_name, account_name, username, **kwargs):
+    return client.delete(resource_group_name=resource_group_name, account_name=account_name, username=username)
+
+
+def list_local_user(client, resource_group_name, account_name, **kwargs):
+    return client.list(resource_group_name=resource_group_name, account_name=account_name)
+
+
+def show_local_user(client, resource_group_name, account_name, username, **kwargs):
+    return client.get(resource_group_name=resource_group_name, account_name=account_name, username=username)
+
+
+def list_keys_for_local_user(client, resource_group_name, account_name, username, **kwargs):
+    return client.list_keys(resource_group_name=resource_group_name, account_name=account_name, username=username)
+
+
+def regenerate_password_for_local_user(client, resource_group_name, account_name, username, **kwargs):
+    return client.regenerate_password(resource_group_name=resource_group_name, account_name=account_name, username=username)
```

Comment:
If it just calls the SDK APIs directly, why not use `storage_command_oauth` instead of `storage_custom_command_oauth`, we don't need to duplicate code in `custom.py`